lib/Diagnostics/Diagnostics.js

```diff
@@ -224,8 +225,8 @@ class Diagnostics extends EventEmitter {
         // Check should be performed only when diagnostics object is updated
         if (this.lastDiagnosticsObjTime === metaData.source.currentTime)
             return;
-
-        this._diagnosticsAnalyzer.runFullDiagnosticsAnalysis(metaData);
+        // beaconPrefix is the string casting of the server type
```

Comment:
i think it's better to pass entryObject so you have this info instead of passing multiple args

Reply:
done